common/djangoapps/embargo/test_utils.py

```diff
@@ -10,7 +10,7 @@
 
 
 @contextlib.contextmanager
-def restrict_course(course_key, access_point="enrollment"):
+def restrict_course(course_key, access_point="enrollment", access_check=False):
```

Comment:
This variable name might be clearer if you called it `disable_access_check`, since that's how it's used below.